hadoop-ozone/dev-support/checks/kubernetes.sh

```diff
@@ -13,9 +13,37 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 
+if [[ "$(uname -s)" = "Darwin" ]]; then
+  echo "k3s is not supported on Mac" >&2
+  exit 1
+fi
+
+export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
```

Comment:
```suggestion
: ${KUBECONFIG:=/etc/rancher/k3s/k3s.yaml}
```

Just to support minikube and other users....